### PR TITLE
Use agent image from docker hub

### LINKED ISSUE #41: Use cri node-agent image instead of build one in Makefile
node agent image is available here:
https://hub.docker.com/r/intel/cri-resmgr-agent

### PR DESCRIPTION
**Pull request by ppalucki**
resolves #41

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -48,10 +48,12 @@
...
+        "resmgrsockets",
         "resourcemanagerv",
         "sapcloud",
         "testbroken",
         "unmarshalled",
         "unmarshalling"
-    ]
+    ],
+    "cSpell.enabled": true
 }
```
Comment (mmucek95) [inline .vscode/settings.json:59]: Missing new line at the end of the file.

File: Makefile [hunk 2]
```diff
@@ -68,19 +70,13 @@ _install-binaries:
...
+_build-installation-image:
 	docker build -t $(REGISTRY)$(INSTALLATION_IMAGE_NAME):$(TAG) -f Dockerfile --target $(INSTALLATION_IMAGE_NAME) .
 
+build-images: _build-extension-image _build-installation-image
+
 push-images:
 	docker push $(REGISTRY)$(EXTENSION_IMAGE_NAME):$(TAG)
-	docker push $(REGISTRY)$(INSTALLATION_IMAGE_NAME):$(TAG)
-	docker push $(REGISTRY)$(AGENT_IMAGE_NAME):$(TAG)
+	docker push $(REGISTRY)$(INSTALLATION_IMAGE_NAME):$(TAG)
```
Comment (mmucek95) [inline Makefile:82]: Missing new line at the end of the file.

File: hacks/disable-extension-in-local-shoot.sh [hunk 3]
```diff
@@ -0,0 +1,2 @@
+set -x
```
Comment (mmucek95) [inline hacks/disable-extension-in-local-shoot.sh:1]: Missing copyright notice

File: hacks/enable-extension-in-local-shoot.sh [hunk 4]
```diff
@@ -0,0 +1,2 @@
+set -x
```
Comment (mmucek95) [inline hacks/enable-extension-in-local-shoot.sh:1]: Missing copyright notice

File: hacks/reconcile-extension-trigger.sh [hunk 5]
```diff
@@ -0,0 +1 @@
+kubectl -n shoot--local--local annotate extension cri-resmgr-extension "gardener.cloud/operation=reconcile" --overwrite
```
Comment (mmucek95) [inline hacks/reconcile-extension-trigger.sh:1]: Missing copyright notice

**Review by mmucek95 — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (ppalucki) [reply .vscode/settings.json:59]: The is auto generated file - and its fine without it ;)

**Review by mmucek95 — APPROVED** [review]

**Pull request merged**